tests/app/celery/test_contact_information_tasks.py

```diff
@@ -53,7 +53,10 @@ def test_should_get_email_address_and_update_notification(client, mocker, notifi
     lookup_contact_info(notification.id)
 
     mocked_get_notification_by_id.assert_called()
-    mocked_va_profile_client.get_email.assert_called_with(EXAMPLE_VA_PROFILE_ID)
+    mocked_va_profile_client.get_email.assert_called_with(mocker.ANY)
+    recipient_identifier = mocked_va_profile_client.get_email.call_args[0][0]
+    assert isinstance(recipient_identifier, RecipientIdentifier)
+    assert recipient_identifier.id_value == EXAMPLE_VA_PROFILE_ID
```

Comment:
because the new functions require a `RecipientIdentifier` as an argument, instead of a VA Profile ID, minor updates to the tests in this file